libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -35,124 +35,10 @@ extern const AP_HAL::HAL &hal;
 const AP_Param::GroupInfo AP_Proximity::var_info[] = {
     // 0 is reserved for possible addition of an ENABLED parameter
 
-    // @Param: _TYPE
-    // @DisplayName: Proximity type
-    // @Description: What type of proximity sensor is connected
-    // @Values: 0:None,7:LightwareSF40c,2:MAVLink,3:TeraRangerTower,4:RangeFinder,5:RPLidarA2,6:TeraRangerTowerEvo,8:LightwareSF45B,10:SITL,12:AirSimSITL,13:CygbotD1
-    // @RebootRequired: True
-    // @User: Standard
-    AP_GROUPINFO_FLAGS("_TYPE",   1, AP_Proximity, _type[0], 0, AP_PARAM_FLAG_ENABLE),
-
-    // @Param: _ORIENT
-    // @DisplayName: Proximity sensor orientation
-    // @Description: Proximity sensor orientation
-    // @Values: 0:Default,1:Upside Down
-    // @User: Standard
-    AP_GROUPINFO("_ORIENT", 2, AP_Proximity, _orientation[0], 0),
-
-    // @Param: _YAW_CORR
-    // @DisplayName: Proximity sensor yaw correction
-    // @Description: Proximity sensor yaw correction
-    // @Units: deg
-    // @Range: -180 180
-    // @User: Standard
-    AP_GROUPINFO("_YAW_CORR", 3, AP_Proximity, _yaw_correction[0], 0),
-
-    // @Param: _IGN_ANG1
-    // @DisplayName: Proximity sensor ignore angle 1
-    // @Description: Proximity sensor ignore angle 1
-    // @Units: deg
-    // @Range: 0 360
-    // @User: Standard
-    AP_GROUPINFO("_IGN_ANG1", 4, AP_Proximity, _ignore_angle_deg[0], 0),
```

Comment:
I think sensor specific could be required because the sensors could easily be placed in different locations on the vehicle.  So for example a 360 lidar on the front and back of a vehicle.  The forward facing one might need to ignore everything backwards (because this would just be the vehicle) while the other needs to ignore everything forwards.  This was my thinking anyway.